locations/spiders/norisko_fr.py

```
@@ -0,0 +1,30 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class NoriskoFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "norisko_fr"
+    item_attributes = {"brand": "Norisko", "brand_wikidata": "Q141159455"}
+    sitemap_urls = ["https://www.dekra-norisko.fr/sitemap.xml"]
+    sitemap_rules = [(r"https://www\.dekra-norisko\.fr/norisko/controle-technique/[^?]+", "parse_sd")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)